jme3-core/src/main/java/com/jme3/anim/AnimLayer.java

```diff
@@ -306,4 +308,20 @@ public Object jmeClone() {
             throw new AssertionError();
         }
     }
+
+    @Override
+    public void write(JmeExporter ex) throws IOException {
+        OutputCapsule oc = ex.getCapsule(this);
+        oc.write(name, "name", null);
+        if(mask instanceof Savable) {
```

Comment:
Please add an extra space between `if` and `(` in the code, following the JME style guide.

```java
if (...
```